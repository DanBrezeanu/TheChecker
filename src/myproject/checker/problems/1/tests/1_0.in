-758573584
0
