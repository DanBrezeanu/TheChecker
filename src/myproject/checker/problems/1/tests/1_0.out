-758573584
-758573584
